README.md

```diff
@@ -1 +1,92 @@
-# bahmni-metabase
\ No newline at end of file
+# bahmni-metabase
+
+## Table of Contents
+* [Metabase Overview](#metabase-overview)
+* [Admin user creation for Metabase](#admin-user-creation-for-metabase)
+* [Connecting Metabase with Database](#connecting-metabase-with-database)
+* [Configure Predefined Reports in Metabase](#configure-predefined-reports-in-metabase)
+* [Starting Metabase from Bahmni Docker](#starting-metabase-from-bahmni-docker)
+
+
+# Metabase Overview
+
+Metabase is an open-source business intelligence tool. Metabase lets you ask questions about your data, and displays answers in formats that make sense, whether that’s a bar chart or a detailed table.
+
+You can save your questions, and group questions into handsome dashboards. Metabase also makes it easy to share questions and dashboards with the rest of your team.
+
+As a part of our initiative we have used [v0.44.6](https://hub.docker.com/layers/metabase/metabase/v0.44.6/images/sha256-527a30f88f79a90bba0951d352a9fcbdecbf5f1b011a41b30e7a074265549168?context=explore) version of Metabase image.
+
+# Admin user creation for Metabase
+The first user of metabase will be created as an Admin user and is pre-configured for [metabase profile](https://github.com/Bahmni/bahmni-docker/blob/master/bahmni-standard/docker-compose.yml) in the docker-compose.yml of bahmni-docker by adding the following configurational data for admin.
+
+MB_ADMIN_EMAIL
+
+MB_ADMIN_FIRST_NAME
+
+MB_ADMIN_PASSWORD
+
+
+# Connecting Metabase with Database
```

Comment:
I am feeling a bit confused. The documentation of Admin User Creation, mentions settings values in docker-compose.yml, which is basically, used AFTER docker image creation, and is a good place to specify, because this is where people will "consume" the metabase image. 

But if I see the documentation of this section, it talks about setting DB creds in current repo, which is BEFORE docker image creation. I think we should make it clear that this is only for REFERENCE PURPOSES, and not the way people will use Metabase-DB connection in general. For that, they are expected to go via the UI for now. Isn't it?

Reply:
I see the confusion here. 
We will update the documents accordingly